lib/as_json_presentable/presenter.rb

```diff
@@ -31,6 +31,11 @@ def as_json(options=nil)
       end
     end
 
+   # return object errors
+  def as_error_json(options=nil)
+    { errors: @resource.errors }
```

Comment:
Can you drop the `@` symbol? There's an `attr_reader`, so you can just call `resource.errors` instead of accessing the instance variable directly.